front/app/components/Form/Components/Layouts/PageControlButtons.tsx

```diff
@@ -35,79 +34,61 @@ const PageControlButtons = ({
   showSubmit,
   dataCyValue,
 }: Props) => {
-  const modalPortalElement = document.getElementById('modal-portal');
   const theme = useTheme();
   const isSmallerThanPhone = useBreakpoint('phone');
 
-  return modalPortalElement
-    ? createPortal(
-        <Box
-          display="flex"
-          flexDirection="row"
-          justifyContent="center"
-          w="100%"
-          zIndex="1010"
-          position="fixed"
-          borderRadius="2px"
-        >
-          <Box
-            position="fixed"
-            bottom={isSmallerThanPhone ? '0px' : '40px'}
-            maxWidth="700px"
-            width="100%"
-            display="flex"
-            justifyContent="space-between"
-            alignItems="center"
-            bgColor={colors.white}
-            borderTop={`1px solid ${colors.borderLight}`}
-            padding={isSmallerThanPhone ? '16px' : '16px 24px'}
```

Comment:
I took out any styles related to this component's positioning to the parent. It is much clearer to have all that sort of styles in the parent and to not mix it up with styles used for other purposes.